server/sockets/list.js

```diff
@@ -107,11 +109,25 @@ const updateItemState = (socket, itemClientLocks) => {
 
 const updateItem = socket => {
   socket.on(ItemActionTypes.UPDATE_SUCCESS, data => {
-    const { listId } = data;
+    const { listId, userId } = data;
+    const sanitizedUserId = sanitize(userId);
+    let editedBy;
+
+    User.findById(sanitizedUserId)
+      .then(user => {
+        if (user) {
+          const { displayName } = user;
+          editedBy = displayName;
+        }
+      })
+      .then(() => {
```

Comment:
why this `then`? shouldn't all that code be in `if` block of previous `then`? What happens if user is not found?

Reply:
When the item is updated, there is ALWAYS a User that is performing that action, so the user will always be found.

Reply:
I think the thens should be combined so it is more clear now - `let` declaration can be moved inside.